webapps-backup/src/main/java/io/camunda/webapps/backup/repository/BackupRepositoryProps.java

```diff
@@ -29,5 +42,7 @@ static Long defaultIncompleteCheckTimeoutInSeconds() {
     return 5 * 60L;
   }
 
-  boolean includeGlobalState();
+  default boolean includeGlobalState() {
```

Comment:
this flag=true was needed by tasklist

Reply:
:question: Why make this a default though? I'd prefer no default implementation and making this explicit for both Tasklist and Operate when you return the anonymous types implementing `BackupRepositoryProps`.

:question: Your changes overall mean that now both Operate and Tasklist have this as `true` whereas before it was only true for Tasklist. Is this intended?

Reply:
Because i'm not even sure if it should be configurable. Now there is no operate vs tasklist anymore, just "webapps" indices. 
By default (in ES/OS) is true, but it was set as false in operate. So in the end we could even get rid of this setting

Reply:
Okay makes sense I guess :+1: